resources/js/processes/components/ProcessesListing.vue

```diff
@@ -70,6 +70,15 @@
               >
                 <i class="fas fa-cog fa-lg fa-fw"></i>
               </b-btn>
+              <b-btn
+                      variant="link"
+                      @click="onAction('view-documentation', props.rowData, props.rowIndex)"
+                      v-b-tooltip.hover
+                      :title="$t('Documentation')"
```

Comment:
Not sure if this was discussed elsewhere and I missed it, but it seems, according to the issue, that this should say "View Documentation".

Reply:
Good catch, thanks